errai-ui/src/main/java/org/jboss/errai/ui/rebind/chain/TemplateCatalog.java

```diff
@@ -86,4 +85,101 @@ public Document parseTemplate(URL template) {
   Chain getChain() {
     return chain;
   }
+
+  private static Document jsoup2DOM(org.jsoup.nodes.Document jsoupDocument) {
```

Comment:
Could you please add a comment here documenting that these methods were copied from Apache Stanbol?